OneCommunityGlobal/HighestGoodNetworkApp
# Peterson implement default error page to be created and redirected to

# Description
This pull request has been opened to implement an error page after the user uses a link to redirect to a page that does not exist.

## Related PRS (if any):
None

## Main changes explained:
Six components have been modified to implement the page of error.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Log in as any user.
4. Go to a URL that does not exist, e.g., http://localhost:3000/project
5. If the user is logged in, two links should appear on the error page: the first redirects to the dashboard, and the second also redirects to the same route but should open the modal for suggestions and development.
6. If the user is not logged in, a link should appear on the error page that redirects to the login page.

## Screenshots or videos of changes:
![Peterson_implement_Default_Error_Page_to_be_created_and_redirected_to](https://github.com/user-attachments/assets/9c1bfb10-7524-44f5-8655-630937430265)

## Note:
None
